src/cockpit/channel.py

```diff
@@ -95,6 +97,9 @@ def do_close(self):
     def do_pong(self, message):
         pass
 
+    def do_options(self, message):
+        pass
```

Comment:
how about `raise ChannelError("this channel don't do options")`?

Reply:
Sounds fine!